lib/processors/bundlers/flexChangesBundler.js

```diff
@@ -34,10 +85,10 @@ module.exports = function({resources, options}) {
 		log.info("Changes collected. Number of changes: " + nNumberOfChanges);
 		const result = [];
 		if (nNumberOfChanges > 0) {
-			changesContent = sortAndStringify(changesContent);
+			changesContent = sortAndStringifyInFlexFormat(changesContent);
 			result.push(resourceFactory.createResource({
-				path: `${options.pathPrefix}/changes/changes-bundle.json`,
-				string: "[" + changesContent.join(",") + "]"
+				path: `${options.pathPrefix}/changes/` + bundleName,
```

Comment:
For consistency, I would include the new variable into the template string:

  .../changes/` + bundleName    -->     .../changes/${bundleName}`

Reply:
done